.github/workflows/cmake.yml

```diff
@@ -0,0 +1,78 @@
+name: CMake
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
```

Comment:
Do we want to support other Windows/Linux other than latest? What's the earliest version of these two that we support? Can we test under those, too?

Reply:
Thanks for looking into this. Shall we track that as an issue in this repo?